db_test.go

```diff
@@ -948,26 +948,29 @@ func TestSizeBasedRetention(t *testing.T) {
 
 	// Add some empty blocks.
 	blocks := []*BlockMeta{
-		{ULID: ulid.MustNew(100, nil), MinTime: 100, MaxTime: 200},
-		{ULID: ulid.MustNew(200, nil), MinTime: 200, MaxTime: 300},
-		{ULID: ulid.MustNew(300, nil), MinTime: 300, MaxTime: 400},
-		{ULID: ulid.MustNew(400, nil), MinTime: 400, MaxTime: 500},
-		{ULID: ulid.MustNew(500, nil), MinTime: 500, MaxTime: 600},
+		{MinTime: 100, MaxTime: 200},
+		{MinTime: 200, MaxTime: 300},
+		{MinTime: 300, MaxTime: 400},
+		{MinTime: 400, MaxTime: 500},
+		{MinTime: 500, MaxTime: 600},
 	}
 
-	var emptyBlockSize int64
+	var singleBlockSize int64
 	for _, m := range blocks {
```

Comment:
Do you remember why/if there's any reason for doing this in a loop? Doesn't really make much sense to me

Reply:
the `m.MinTime, m.MaxTime` are different for each block.

you can probably even do something like:
```
for _, m := range int[100,200,300,400] {
	singleBlockSize = createPopulatedBlock(t, db.Dir(), 100, m, m+100).Size()
}
```

but what we have now is easier to read.

Reply:
But it just keeps overwriting singleBlockSize each time, it doesn't do anything with that value until after the loop. It's effectively the same thing as 
```
singleBlockSize = createPopulatedBlock(t, db.Dir(), 100, 500, 600).Size()
```
without being in a loop at all.

Reply:
yeah that is not important, Just save few extra lines of code to have it write to it once.